magic-eight-ball/src/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom";
+import "./styles.css";
+
+const API_ENDPOINT = 'https://yesno.wtf/api';
+
+function App() {
+  const [question, setQuestion] = useState("");
+  const [item, setItem] = useState("");
+  const [error, setError] = useState(false);
+  const [isLoaded, setIsLoaded] = useState(false);
+
+  function handleOnChange(event) {
+    setQuestion(event.target.value);
+  }
+
+  function submitting() {
+    if (question.endsWith('?')) {
+      setIsLoaded(true);
+      setError(false);
+      setQuestion(question);
+      fetch(API_ENDPOINT).then((res) => res.json()).then((data) => setItem(data))
+    }
+    else {
+      setError(true);
```

Comment:
looks like we don't need to separate states for `isLoaded` and `error`. You can reduce to one

Reply:
I couldn't see any change related to this conversation

Reply:
we talked yesterday that we can't do it with a single variable